primo/data_parser/default_data.py

```diff
@@ -297,33 +297,45 @@ class _SupportedContent:
     ),
 }
 
+# note the names of these metrics must be identical to
+# a property of the Optimal Project class for the
+# efficiency calculation
 SUPP_EFF_METRICS = {
     "num_wells": _SupportedContent(
-        name="num_wells",
-        full_name="Number of Wells",
+        name="num_wells", full_name="Number of Wells", required_data="well_id"
     ),
     "num_unique_owners": _SupportedContent(
         name="num_unique_owners",
         full_name="Number of Unique Owners",
+        required_data="operator_name",
+        has_inverse_priority=True,
     ),
-    "dist_centroid": _SupportedContent(
-        name="dist_centroid",
-        full_name="Distance to Centroid [miles]",
-    ),
-    "elevation_delta": _SupportedContent(
-        name="elevation_delta",
+    "avg_elevation_delta": _SupportedContent(
```

Comment:
Now, we plan to use infinity norm for both combined objective and post-optimization efficiency calculations. So should we change it back to `elevation_delta`? Also, please add the `fill_missing_value` attribute for data processing:
`fill_missing_value={"domain": NonNegativeFloat, "default": 0},`. @DevKakkad32 @apd-ypuranik What is the appropriate default value for missing elevation delta information?

Reply:
Holding off on naming for a future PR. Added the fill_missing_value field. Will update it based on the response from @DevKakkad32 and @apd-ypuranik